plugins/hls-eval-plugin/test/cabal.project

```diff
@@ -0,0 +1,3 @@
+packages:
```

Comment:
I moved the `test/testdata/cabal.project` file one directory up, so I could define another library `info-util`. This was because `:info` prints the origin comment (`-- defined in "..."`) using the full path to the file for modules in the same package, which wouldn't be portable.